tensorflow/core/kernels/data/name_utils.cc

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/core/kernels/data/name_utils.h"
+#include "tensorflow/core/platform/logging.h"
+
+namespace tensorflow {
+namespace data {
+namespace name_utils {
+
+string OpName(const string& dataset_type) {
+  if (dataset_type == "Map") {
+    return "MapDataset";
+  } else if (dataset_type == "Prefetch") {
+    return "PrefetchDataset";
+  } else if (dataset_type == "Range") {
+    return "RangeDataset";
+  }
+  LOG(FATAL) << "Unknown dataset type " << dataset_type << std::endl;
+  return "Unknown dataset type";
+}
+
+namespace internal {
```

Comment:
why the internal namespace?

Reply:
In the beginning, I thought this function should not be public, so move it to the internal namespace. Will remove it.